doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -531,24 +465,24 @@ to apply specific treatments accordingly.
 This paragraph describes the basic form of pattern matching. See
 Section :ref:`Mult-match` and Chapter :ref:`extendedpatternmatching` for the description
 of the general form. The basic form of pattern matching is characterized
-by a single :token:`case_item` expression, a :token:`patterns_comma` restricted to a
+by a single :token:`case_item` expression, an :token:`eqn` restricted to a
 single :token:`pattern` and :token:`pattern` restricted to the form
 :n:`@qualid {* @ident}`.
 
-The expression match ":token:`term`:math:`_0` :token:`return_type_opt` with
-:token:`pattern`:math:`_1` => :token:`term`:math:`_1` :math:`|` … :math:`|`
-:token:`pattern`:math:`_n` => :token:`term`:math:`_n` end" denotes a
+The expression :n:`match @term {? return @term100 } with
+@pattern`:math:`_1` :n:`=> @term`:math:`_1` :n:`| … | @pattern`:math:`_n`
+:n:`=> @term`:math:`_n` :n:`end` denotes a
```

Comment:
You could go further and fix the bad quoting practice here.

Reply:
I think the subscripts are not needed.  They're a bit pedantic. Buh-bye

Also odd to use both n and i as indices.

Reply:
Usually `i` is used throughout the manual to denote a variable varying between `0` or `1` and `n`.